### PR TITLE
Fix Issue #67

### PR DESCRIPTION
**Pull request by vedantrathore**
Changed `flag` keyword to `struct`

### CODE REVIEW

**Comment by vedantrathore** [comment]
@brson I think it looks good to merge :)

File: src/lib.rs [hunk 1]
```diff
@@ -230,7 +230,7 @@ macro_rules! bitflags {
                     )+
                 }
 
-                // Conditionally override the check for just those flags that
+                // Conditionally override the check for just those struct that
```
Comment (dtolnay) [inline src/lib.rs]: This should still say flags.

Comment (vedantrathore) [inline src/lib.rs]: Sorry, my bad.

**Comment by vedantrathore** [comment]
@dtolnay done. :)

**Comment by alexcrichton** [comment]
Thanks!

**Pull request merged**